src/System.Diagnostics.PerformanceCounter/ref/System.Diagnostics.PerformanceCounter.cs

```diff
@@ -243,50 +243,50 @@ public sealed class CounterSetInstanceCounterDataSet : IDisposable
-        RateOfCountPerSecond32 = 0x10410400,
-        RateOfCountPerSecond64 = 0x10410500,
-        RawFraction32 = 0x20020400,
-        RawFraction64 = 0x20020500,
-        RawBase32 = 0x40030403,
-        RawBase64 = 0x40030500,
-        SampleFraction = 0x20C20400,
-        SampleCounter = 0x00410400,
-        SampleBase = 0x40030401,
-        AverageTimer32 = 0x30020400,
-        AverageBase = 0x40030402,
-        AverageCount64 = 0x40020500,
-        PercentageActive = 0x20410500,
-        PercentageNotActive = 0x21410500,
-        PercentageActive100Ns = 0x20510500,
-        PercentageNotActive100Ns = 0x21510500,
-        ElapsedTime = 0x30240500,
-        MultiTimerPercentageActive = 0x22410500,
-        MultiTimerPercentageNotActive = 0x23410500,
-        MultiTimerPercentageActive100Ns = 0x22510500,
-        MultiTimerPercentageNotActive100Ns = 0x23510500,
-        MultiTimerBase = 0x42030500,
-        Delta32 = 0x00400400,
-        Delta64 = 0x00400500,
-        ObjectSpecificTimer = 0x20610500,
-        PrecisionSystemTimer = 0x20470500,
-        PrecisionTimer100Ns = 0x20570500,
-        PrecisionObjectSpecificTimer = 0x20670500
+        RawDataHex32 = 0,
+        RawDataHex64 = 256,
+        RawData32 = 65536,
+        RawData64 = 65792,
+        Delta32 = 4195328,
+        Delta64 = 4195584,
+        SampleCounter = 4260864,
+        QueueLength = 4523008,
+        LargeQueueLength = 4523264,
+        QueueLength100Ns = 5571840,
+        QueueLengthObjectTime = 6620416,
+        RateOfCountPerSecond32 = 272696320,
+        RateOfCountPerSecond64 = 272696576,
+        RawFraction32 = 537003008,
+        RawFraction64 = 537003264,
+        PercentageActive = 541132032,
+        PrecisionSystemTimer = 541525248,
+        PercentageActive100Ns = 542180608,
+        PrecisionTimer100Ns = 542573824,
+        ObjectSpecificTimer = 543229184,
+        PrecisionObjectSpecificTimer = 543622400,
+        SampleFraction = 549585920,
+        PercentageNotActive = 557909248,
+        PercentageNotActive100Ns = 558957824,
+        MultiTimerPercentageActive = 574686464,
+        MultiTimerPercentageActive100Ns = 575735040,
+        MultiTimerPercentageNotActive = 591463680,
+        MultiTimerPercentageNotActive100Ns = 592512256,
+        AverageTimer32 = 805438464,
+        ElapsedTime = 807666944,
+        AverageCount64 = 1073874176,
+        SampleBase = 1073939457,
+        AverageBase = 1073939458,
+        RawBase32 = 1073939459,
+        RawBase64 = 1073939712,
+        MultiTimerBase = 1107494144
     }
     public enum CounterSetInstanceType
     {
         Single = 0,
-        Multiple = 0x00000002,
-        GlobalAggregate = 0x00000004,
-        GlobalAggregateWithHistory = 0x0000000B,
-        MultipleAggregate = 0x00000006,
-        InstanceAggregate = 0x00000016
+        Multiple = 2,
+        GlobalAggregate = 4,
+        MultipleAggregate = 6,
+        GlobalAggregateWithHistory = 11,
+        InstanceAggregate = 22
     }
```

Comment:
@adiaaida i converted in integer to maintain style of other enum in this cs. Used simple script to avoid copy/paste errors:

```cs
	foreach (string element in Enum.GetNames(typeof(CounterSetInstanceType)))
	{		
		Enum.TryParse(element, out CounterSetInstanceType ct);
		(element + " = " + ((int)ct)+",").Dump();
	}	
```